Dockerfile

```diff
@@ -3,12 +3,24 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
-# This Dockerfile uses the latest code from the Git repo.
-#   Clone the repo then run 'docker build' with this Dockerfile file to get the latest versions of
-#   *all* CLI modules as in the Git repo.
-
 FROM python:3.5.2-alpine
 
+ARG CLI_VERSION
+
+# Metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+LABEL org.label-schema.schema-version="1.0" \
+      org.label-schema.vendor="Microsoft" \
+      org.label-schema.name="Azure CLI 2.0" \
+      org.label-schema.version=$CLI_VERSION \
+      org.label-schema.license="MIT" \
+      org.label-schema.description="The Azure CLI 2.0 is the new Azure CLI and is applicable when you use the Resource Manager deployment model." \
+      org.label-schema.url="https://docs.microsoft.com/en-us/cli/azure/overview" \
+      org.label-schema.usage="https://docs.microsoft.com/en-us/cli/azure/install-az-cli2#docker" \
+      org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vcs-url="https://github.com/Azure/azure-cli.git" \
+      org.label-schema.docker.cmd="docker run -v ${HOME}:/root -it azuresdk/azure-cli-python:<version>"
+
 WORKDIR azure-cli
 COPY . /azure-cli
```

Comment:
Should we remove this folder from the image after build?

Reply:
The folder would only contain the source code due to the .dockerignore file we have.
Don't see harm in including it.